test.sh

```diff
@@ -74,6 +74,12 @@ openssl req -new -newkey rsa:4096 -days 365 -nodes -x509 \
 kubectl create namespace "$NAMESPACE"
 kubectl -n "$NAMESPACE" create secret tls certificates --cert=cert.crt --key=cert.key
 
+echo 1>&2 "Installing Prometheus Operator"
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+helm install prometheus-operator prometheus-community/kube-prometheus-stack -n "$NAMESPACE"
+kubectl wait --for=condition=Ready pods -l "release=prometheus-operator" -n "$NAMESPACE"
```

Comment:
Thanks for adding this! This and changes in `test-values.yaml` will prove there are no syntax errors in templates, and we can deploy the chart. The only missing piece is a test in `charts/trino/templates/tests/`. The existing `test-connection` runs a query in Trino to verify the basic connectivity. I think we should add a new one, that would query Prometheus and verify the metrics are being collected.